spark/spark-scala-parent/pom.xml

```diff
@@ -34,6 +34,11 @@
     <packaging>pom</packaging>
     <name>Zeppelin: Spark Scala Parent</name>
 
+    <properties>
+        <!-- -->
+        <spark.runtime.version>2.2.0</spark.runtime.version>
```

Comment:
in else where the default  is 2.4.0? should this be also to match?

Reply:
I changed it to 2.4.0, but actually it doesn't matter which version I specified here. Because type of spark-scala-parent is pom and the spark version will be override by its child module scala-2.10 and scala-2.11.